src/assets/locales/es.json

```diff
@@ -1,169 +1,169 @@
 {
   "app-name": "Aplicación",
-  "editor-name": "Editor de escenas",
+  "editor-name": "Editor de escena",
   "contact-email": "app@company.com",
   "company-name": "Empresa",
   "share-hashtag": "#app",
-  "app-description": "Comparte una habitación virtual con amigos.\nMira videos, juega con objetos 3D o simplemente pasa el rato.",
-  "app-tagline": "VR social privada en tu navegador web",
-  "auth.verified-title": "¡Correo electrónico verificado!",
-  "auth.verify-failed": "No se puede iniciar sesión con este enlace. Es posible que ya se haya utilizado o haya caducado.",
-  "auth.verified": "¡Tu correo electrónico ha sido verificado!\nAhora puedes cerrar esta pestaña del navegador y volver a %app-name%.",
-  "auth.spoke-verified": "¡Tu correo electrónico ha sido verificado!\nAhora puedes cerrar esta pestaña del navegador y volver a %editor-name%.",
-  "sign-in.prompt": "Inicia sesión para fijar objetos en las habitaciones.",
-  "sign-in.admin": "Revisa tu correo electrónico para ver si hay un correo electrónico de verificación. Una vez verificado, ingresa tu correo electrónico para crear tu cuenta o iniciar sesión.",
-  "sign-in.admin-no-permission": "No tienes acceso a las herramientas de administración. Inicia sesión en otra cuenta o pídele a un administrador que te otorgue permiso",
-  "sign-in.hub": "Se requiere una cuenta para unirse a las salas.\n\nIntroduce tu correo electrónico para crear tu cuenta o iniciar sesión.",
-  "sign-in.auth-started": "¡Correo electrónico enviado a {email}!\n\nPara continuar, haz clic en el enlace del correo electrónico con tu teléfono, tableta o PC.\n\n¿No tienes correo electrónico? En ese caso quizá no puedas crear una cuenta.",
-  "sign-in.pin": "Debes iniciar sesión para anclar objetos",
+  "app-description": "Comparte una sala virtual con amigos.\nMira videos, juega con objetos 3D o simplemente pasa el rato.",
+  "app-tagline": "VR social privado en tu navegador web",
+  "auth.verified-title": "¡Correo verificado!",
+  "auth.verify-failed": "Imposible registrarse con este vínculo. Puede que ya haya sido usado o haya expirado.",
+  "auth.verified": "¡Tu correo ha sido verificado!\nAhora puedes cerrar esta pestaña y volver a %app-name%.",
+  "auth.spoke-verified": "¡Tu correo ha sido verificado!\nAhora puedes cerrar esta pestaña y volver a %editor-name%.",
+  "sign-in.prompt": "Inicia sesión para fijar objetos en salas.",
```

Comment:
"Inicia sesión para fijar objetos en salas." should be "Inicia sesión para fijar objetos en las salas."